src/schema/paddle.ts

```diff
@@ -231,6 +362,73 @@ export const resolvers: IResolvers<unknown, AuthContext> = traceResolvers<
         items,
       };
     },
+    plusPricingMetadata: async (
+      _,
+      { variant = DEFAULT_PLUS_METADATA }: PlusMetadataArgs,
+      ctx: AuthContext,
+    ): Promise<PlusPricingMetadata[]> =>
+      await getPlusPricingMetadata(ctx.con, variant),
```

Comment:
```suggestion
      getPlusPricingMetadata(ctx.con, variant),
```

Since we're just returning it directly, we don't need the `await` here.

Reply:
Kinda helps with error messages normally from what I've experienced, but yeah, we can definitely just remove this if the default error message would be helpful already.